cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -180,6 +186,15 @@ static Probe tlsSidecarLivenessProbe(TlsSidecar tlsSidecar) {
         return createExecProbe(Arrays.asList("/opt/stunnel/stunnel_healthcheck.sh", "2181"), tlsSidecarLivenessProbe);
     }
 
+    protected static Probe kafkaJmxMetricsBrokerProbe(io.strimzi.api.kafka.model.Probe  kafkaJmxMetricsReadinessProbe, String clusterName) {
```

Comment:
Out of curiosity, what is the reason to have this here when it is specific to the JmxTrans deployment? Why not keep it in its own class?

Reply:
I was just following the `TLS_SIDECAR` as an example but you are right will move it to it's own class!